COMMERCIAL_LICENSE_AGREEMENT.txt

```diff
@@ -1,6 +1,6 @@
 COMMERCIAL LICENSE AGREEMENT
 
-Software Name: HungerMapLive CSEE
+Software Name: CaochAI CSEE
```

Comment:
CoachAI should be right 😀 Also feel free to push stuff like this directly without creating a PR, no worries